### PR TITLE
[JFG-925] Changed pom.xml to deploying docker images via maven.

### PR DESCRIPTION
**Pull request by Nmishin**
Dockerfiles for jaas and webclient have been changed also for using ${project.version} env instead hardcoded version.

For build images need to specify profile: mvn install -Pdocker
For build and push images to repository: mvn deploy -Pdocker

"LABEL" has been added to the Dockerfiles for increase verbosity. 
You can watch it: docker inspect griddynamics/jagger-jaas:1.2.6-SNAPSHOT

### CODE REVIEW

File: docker/jagger/jaas/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
 ADD run.sh /com/griddynamics/jagger/run.sh
 RUN chmod +x /com/griddynamics/jagger/run.sh
 
 WORKDIR /com/griddynamics/jagger
 
 CMD /com/griddynamics/jagger/run.sh
+
+LABEL   project.groupId="${project.groupId}" \
+        project.artifactId="${project.artifactId}" \
```
Comment (dlatnikov) [inline docker/jagger/jaas/Dockerfile]: where are project.groupId & project.artifactId used?

Comment (Nmishin) [inline docker/jagger/jaas/Dockerfile]: not used anywhere have been added to be more informative

Comment (dlatnikov) [inline docker/jagger/jaas/Dockerfile]: let's remove if they are not used

Comment (Nmishin) [inline docker/jagger/jaas/Dockerfile]: ok

**Pull request merged**